packages/angular/tslint.json

```
@@ -110,8 +116,8 @@
       "check-decl",
       "check-operator",
       "check-separator",
-	  "check-module",
+      "check-module",
       "check-type"
     ]
   }
-}
+}
```

Missing newline at the end of file.
```suggestion
}

```